gordon/schema/definitions/record-action.json

```diff
@@ -0,0 +1,36 @@
+{
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-03/schema",
+
+  "title": "An action to perform on a DNS record.",
+  "description" : "Schema for creating or deleting selected DNS records.",
+  "properties": {
+    "name": {
+      "type": "string",
+      "required": true,
+      "examples": [
+        "fully-qualified-instance-name-1234.subdomain.example.com",
+        "www.example.com"
+      ]
+    },
+    "target": {
```

Comment:
Will there be another message schema type for actions we don't yet have the information? I assume that a potential message would be like, "instance group created" but we just know the IGM name, not the hosts.

Reply:
Or, maybe to accommodate this, the target should not be required.

Reply:
Actually yeah, especially if it's a delete action, right?